app/upgrades/mainnet/v5/upgrade.go

```diff
@@ -5,19 +5,13 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-
-	liquiditykeeper "github.com/crescent-network/crescent/v4/x/liquidity/keeper"
-	liquiditytypes "github.com/crescent-network/crescent/v4/x/liquidity/types"
 )
 
 const UpgradeName = "v5"
 
 func UpgradeHandler(
-	mm *module.Manager, configurator module.Configurator,
-	liquidityKeeper liquiditykeeper.Keeper) upgradetypes.UpgradeHandler {
+	mm *module.Manager, configurator module.Configurator) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-		liquidityKeeper.SetMaxNumMarketMakingOrdersPerPair(ctx, liquiditytypes.DefaultMaxNumMarketMakingOrdersPerPair)
```

Comment:
Is there a reason for deleting these lines? 
`paramSpace.Set()` and `Set initial value` are independent actions, so we should do both.